makejdk.sh

```diff
@@ -75,6 +75,9 @@ while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
 
     "--jtreg" | "-j" )
     JTREG=true; shift;;
+    
+    "--jtreg_subsets" )
+    JTREG=true; JTREG_TEST_SUBSETS="$1"; shift;;
```

Comment:
Should JTREG_TEST_SUBSETS default to 'the default *before* having $1 (if $1 exists / is non empty) assigned to it?  That way you may not need to have the separate --jtreg_subsets flag?